### DISCUSSION TITLE
Access to other services on server hosting VPN, howto?

### DISCUSSION THREAD

**Discussion by scottty27** [qdm12/gluetun#1402] [Q&A]
I also run other services on a server which I connect to via OpenVPN with Gluetun, for example an SSH server, but I cannot access anything running on the same IP as the OpenVPN server in a container connected through gluetun. I assume this is something to do with the firewall rules that gluetun sets up - please can someone point me in the right direction as to how to allow access to my VPN server IP via gluetun-connected containers? 

For example - connected to openvpn server x.x.x.x - cannot connect to SSH server on same server x.x.x.x when running in a gluetun connected container.

**Comment by qdm12** [comment]
Hello sorry for the massive delay;

So you want `container X` <-> `gluetun` <-> `openvpn server` <-> `another server on the same machine`

Gluetun allows everything to go through the vpn tunnel, so this is likely a problem on the openvpn server machine (firewall, listening address of the other server program)

**Reply by scottty27** [reply]
Hi, no problem, thanks for replying.

Yes, basically the openvpn server that I am connecting to also runs a SSH server, but I can't access the SSH server from a docker container connected to the gluetun network. I don't think its a config issue with the openvpn server as when I connect to the same openvpn server from my mac, I can still access the ssh server via the VPN. I could be wrong though, unfortunately I can't change any of the openvpn server settings (its running on a whatbox.ca seedbox).
